docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "K8ssandra site engineer quick start"
+linkTitle: "Site engineers"
+weight: 2
+description: |
+  Take a tour of your K8ssandra cluster using all the utilities!
+---
+
+**Completion time**: **10 minutes**.
+
+* [Upgrading K8ssandra with Ingress support]({{< relref "#configure-ingress" >}}): Enabling access to K8ssandra from outside the K8s cluster via Traefik.
+* [Accessing K8ssandra utilities]({{< relref "#access-k8ssandra-utilities" >}}): Accessing useful utilities like the Cassandra Reaper repair tool and Grafana metrics reporting.
+* [Starting and stopping K8ssandra]({{< relref "#cassandra-operations" >}}): Cleanly stopping and restarting the K8ssandra pod.
+
+## Configure Ingress
+
+Right now, you can only interact with your Cassandra node within the K8s cluster using `kubectl` commands which is a pretty severe limitation. To enable external access for applications as well as enable access to K8ssandra features like Grafana dashboards, the Reaper Repair UI, and the Stargate GraphQL playground, you'll need to configure Ingress. In this section we'll upgrade your existing K8ssandra installation to support Ingress using a Traefik Helm chart.
```

Comment:
Second sentence is pretty long, could probably use an additional comma: "To enable external access for applications, as well as..."

Reply:
I'm going to say it's not... obnoxiously long. Comma added.